History.md

```diff
@@ -4,6 +4,8 @@
   [Issue #9196](https://github.com/meteor/meteor/issues/9196)
   [PR #9198](https://github.com/meteor/meteor/pull/9198)
 
+* Allow Npm.depends to specify any http or https URL [#9236](https://github.com/meteor/meteor/issues/9236)
```

Comment:
Please move the issue link to a new line, and prefix it with "Issue" (to line up with the other `History.md` entries).

Reply:
Fixed